src/components/MovieDetail.svelte

```diff
@@ -30,8 +30,7 @@
     });
 </script>
 
-<div class="movie-detail" role="button" on:click={backToGallery} in:scale={{ duration: 500 }}
-     out:scale={{ duration: 500 }}>
+<div class="movie-detail" role="button" tabindex="0" on:click={backToGallery} on:keydown={handleKeydown} in:scale={{ duration: 500 }} out:scale={{ duration: 500 }}>
```

Comment:
Instead of using `in:scale` and `out:scale` there is also a `transition:scale` ✨

Reply:
I tried making the moviedetail page zoom in and out from the position the movie was located in the grid i couldn't get this to work properly so i decided to remove this. But thanks for the tip!